### PR TITLE
database set up

### PR DESCRIPTION
**Pull request by romchik2612**

### CODE REVIEW

**Review by Nattalli — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -1,2 +1,7 @@
 SECRET_KEY=ENTER_THE_SECRET_KEY
 DEBUG=1
+DB_NAME=wrfumllk
+DB_USER=wrfumllk
+DB_PASSWORD=cPj0ayCvwHTPtHYDzAC-U0DW6I38orjP
```
Comment (antos07) [inline .env.sample:5]: It isn't a good idea to put real credentials into the repository

**Review by antos07 — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 1 above)
Comment (romchik2612) [reply .env.sample:5]: agree, but don`t think we can put them somewhere else now

Comment (antos07) [reply .env.sample:5]: Our telegram group would be a better place, I guess (safer at least)

Comment (Nattalli) [reply .env.sample:5]: Agree, in the `.env.sample` should be fake data that shows what should be added to the `.env` file

Comment (Nattalli) [reply .env.sample:5]: if it is real data - we should replace it with fake data and separate it to the telegram or Google Sheets, etc.

**Review by Nattalli — APPROVED** [review]

**Review by antos07 — APPROVED** [review]

**Pull request merged**